lib/utils/transcription-utils.js

```diff
@@ -104,6 +105,9 @@ const stickyVars = {
   soniox: [
     'SONIOX_PROFANITY_FILTER',
     'SONIOX_MODEL'
+  ],
+  assemblyai: [
+    'ASSEMBLYAI_API_KEY'
```

Comment:
word boost should cleared as well I think

Reply:
added now